CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # next
 
 * Fixed the perk selection in Loadout Builder. #1453
+* Integrated Trials-centric weapon reviews (and the ability to rate your own gear (and make comments about your gear)).  Done in conjunction with destinytracker.net.
 
```

Comment:
It's destinytracker.com ;). Unless you used it way back in 2014 when it still was destinytracker.net.

Also you're saying gear, might be an idea to change that to weapon?